EIPS/eip-6963.md

```diff
@@ -77,7 +78,7 @@ ipfs://QmZo7gsostaUdrV1peFz4cX6Z2TgriFJJP4VahG5knVm29
 https://ethereum.org/static/f541df14fca86543040c113725b5bd1a/99bcf/metamask.webp
 ```
 
-Additionally the image must be squared with 96x96px minimum resolution. Image format is recommended to be lossless like PNG and WebP or alternatively vectorial like SVG. We strongly discourage lossy formats like JPG/JPEG.
+The `icon` string MUST be a data URI as defined in [RFC 2397]. The image SHOULD be squared with 96x96px minimum resolution. The image format is RECOMMENDED to be either lossless or vector based such as PNG, WebP or SVG to make the image easy to render on the DApp. Since SVG images can execute Javascript, applications and libraries MUST render SVG images using the `<img>` tag to ensure no untrusted Javascript execution can occur. The DApp SHOULD reject providers from being an available option with an icon using a lossy formats such as JPG or JPEG in order to prevent pixelation of images.
```

Comment:
Is there any other reason besides pixelation to reject a provider for presenting a lossy format?

```suggestion
The `icon` string MUST be a data URI as defined in [RFC 2397]. The image SHOULD be squared with 96x96px minimum resolution. The image format is RECOMMENDED to be either lossless or vector based such as PNG, WebP or SVG to make the image easy to render on the DApp. Since SVG images can execute Javascript, applications and libraries MUST render SVG images using the `<img>` tag to ensure no untrusted Javascript execution can occur.
```

Reply:
Not that I could think of. This is more about just eliminating unnecessary code branches, but I don't think this is likely to cause issues for us anyways.